README.md

```diff
@@ -3,6 +3,10 @@
 [![Code Climate](https://codeclimate.com/github/fabric8io/fluent-plugin-kubernetes_metadata_filter/badges/gpa.svg)](https://codeclimate.com/github/fabric8io/fluent-plugin-kubernetes_metadata_filter)
 [![Test Coverage](https://codeclimate.com/github/fabric8io/fluent-plugin-kubernetes_metadata_filter/badges/coverage.svg)](https://codeclimate.com/github/fabric8io/fluent-plugin-kubernetes_metadata_filter)
 
+This plugin uses the input source to determine basic metadata to associate with the record.  When `kubernetes_url` is configured, the plugin attempts to fetch additional metadata about the pod
+and its associated namespace.  The plugin will orphan a record if it unable to determine the pod or namespace metadata.  The namespace will be set to `orphaned_namespace_name` and the recored property
```

Comment:
Either it should not orphan the record if it cannot determine the pod metadata, or that behavior should be configurable.  For our (o-a-l) use case, we only want to orphan records if we cannot determine the namespace metadata - the pod metadata is not so important.

Reply:
`recorded`

Reply:
I think you mean `record property`?  Maybe use the word `field` (as in a field of the record) instead of `property`?